sharding-core/src/main/java/in/cleartax/dropwizard/sharding/hibernate/MultiTenantHibernateBundle.java

```diff
@@ -35,6 +35,15 @@
     private final MultiTenantSessionFactoryFactory sessionFactoryFactory;
     private SessionFactory sessionFactory;
     private boolean lazyLoadingEnabled = true;
+    private boolean registerHealthCheck = false;
+
+    public boolean isregisterHealthCheck() {
```

Comment:
Please use the camelCased names

Reply:
Got changed mistakenly at the time of replacing the flag name. Updated in new commit.